packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point-input.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import WheelPicker from '../../Form/WheelPicker';
+import { Popover, Text, useDevice } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset';
+import { Localize, localize } from '@deriv/translations';
+import './payout-per-point.scss';
+import { LabelPairedChevronsDownCaptionRegularIcon, LabelPairedChevronsUpCaptionRegularIcon } from '@deriv/quill-icons';
+
+const PayoutPerPointInput = ({
+    payoutOptions,
+    onPayoutClick,
+    selectedBarrier,
+    defaultPayout,
+    currency,
+    tooltipText,
+}: {
+    payoutOptions: number[];
+    onPayoutClick: (option: number) => void;
+    selectedBarrier: string;
+    defaultPayout: number;
+    currency: string;
+    tooltipText?: React.ReactNode;
+}) => {
+    const { is_desktop } = useDevice();
+    const turbos_payout_message = (
+        <Localize i18n_default_text='The ammount you will receive at expiry for every point of change above the barrier.' />
+    );
+    return is_desktop ? (
+        <Fieldset
+            className={'trade-container__fieldset payout-per-point-input'}
+            header={localize('Payout per Point')}
+            header_tooltip={turbos_payout_message}
+            popover_wrapper_class='popover_wrapper_class'
+        >
+            <WheelPicker
+                options={payoutOptions}
+                defaultValue={defaultPayout}
+                onClick={onPayoutClick}
+                currency={currency}
+            />
+            <Fieldset className='actions-wrapper'>
+                <Text size={'xxxs'} line_height='l' color='default' align='center' as='p'>
+                    {localize('Distance to current spot')}
+                </Text>
+                <Popover
+                    alignment='left'
+                    className='popover-icon'
+                    is_bubble_hover_enabled
+                    margin={206}
+                    disable_target_icon
+                    icon='info'
+                    zIndex='9999'
+                    message={tooltipText}
+                >
+                    <div className='distance-to-current-spot'>
+                        <Text
+                            size={'xxxs'}
+                            line_height='l'
+                            color={'default'}
+                            align='center'
+                            as='p'
+                            className='barrier-value'
+                        >
+                            {selectedBarrier}
+                            {Number(selectedBarrier) < 0 ? (
+                                <LabelPairedChevronsDownCaptionRegularIcon
+                                    width={12}
```

Comment:
we should probably be using predefined size instead of setting width and height

Reply:
Not sure there are size for quill-icon . couldn't fine size props

Reply:
The smallest one is <LabelPairedChevronDownSmRegularIcon    /> and <StandaloneChevronDownRegularIcon   fill='#000000' iconSize='xs'/>, but both of them are bigger then 12px
<img width="1524" alt="Screenshot 2024-07-24 at 9 35 27 AM" src="https://github.com/user-attachments/assets/42c74b9e-18e0-4633-a40e-c2a0d8c2a820">
![Screenshot 2024-07-24 at 9 38 05 AM](https://github.com/user-attachments/assets/00ce8098-42b6-45c1-89b0-1bb340b092b9)
I don't think we have another way of handling it, only the way Ahmad did it